scripts/zones/Lufaise_Meadows/mobs/Flockbock.lua

```diff
@@ -1,11 +1,50 @@
 -----------------------------------
 -- Area: Lufaise Meadows
 --   NM: Flockbock
+-- https://www.bg-wiki.com/ffxi/Flockbock
+-- TODO: NM has several possible spawn locations
+-----------------------------------
+mixins = { require('scripts/mixins/rage') }
 -----------------------------------
 local entity = {}
 
+entity.onMobInitialize = function(mob)
+    -- Has enhanced double attack, movement speed, and uses WS more often than normal mobs.
```

Comment:
comment about move speed here need removed

ws'ing more often might not mean storetp. was his tp measured during the fight? sorry I can't scroll through video's to see everything that happens

Reply:
other ways it could go more often: regain, different tp use thresholds, reaction mechanics (unlikely but just listing things here) [savetp](https://www.bg-wiki.com/ffxi/Save_TP), and [conservetp](https://www.bg-wiki.com/ffxi/Conserve_TP)

its is likely storetp or else regain. 

storetp is confirmable to counting up its tp, regain by just holding it without feeding any.